src/common/Page404/Page404.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import "./style.css";
+import Img404 from "./Page404.png";
+
+function Page404() {
+    return(
+        <img className="page404__img" src={Img404} alt="Error 404 info page" />
```

Comment:
Page404 component returns only one element so please consider to use className="page404" only :)